aiokafka/record/_legacy_records.pyx

```diff
@@ -429,7 +440,9 @@ cdef class _LegacyRecordBatchBuilderCython:
                     compressed = lz4_encode(bytes(self._buffer))
             else:
                 return 0
-            size = _size_in_bytes(self._magic, key=None, value=compressed)
+            value_size = _size_of_bytearray(compressed)
```

Comment:
Could you explain why was the change to C extension made? Does it make a speed difference?

Reply:
`_size_in_bytes` was changed to accept lengths rather than objects for the sake of consistency between the Python and C versions. `_size_of_bytearray` was added to avoid reproducing that logic all over the place.

This mattered more in the original version when the signature of `size_in_bytes()` had changed. I'm happy to revert it, if you'd prefer.

Reply:
Yes please